NuGet.config

```diff
@@ -3,8 +3,10 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <!--  Begin: Package sources from dotnet-razor-compiler -->
+    <add key="darc-pub-dotnet-razor-compiler-e463fff" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-razor-compiler-e463ffff/nuget/v3/index.json" />
+    <!--  End: Package sources from dotnet-razor-compiler -->
     <!--  Begin: Package sources from dotnet-aspnetcore -->
-    <add key="darc-pub-dotnet-aspnetcore-717740e" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspnetcore-717740e2/nuget/v3/index.json" />
```

Comment:
/fyi this line isn't needed because dotnet/razor-tooling no longer consumes a shipping package from dotnet/aspnetcore (`dotnet6-transport` is enough for that repo). May need a temporary dotnet/runtime feed after next dotnet/aspnetcore build flows into this repo.